frontend/packages/kubevirt-plugin/src/components/modals/scheduling-modals/shared/LabelRow/label-row.scss

```diff
@@ -0,0 +1,12 @@
+.kv-label__row {
```

Comment:
do we have to use css here? can't we rather use https://patternfly-react.surge.sh/documentation/react/layouts/grid or https://patternfly-react.surge.sh/documentation/react/layouts/split or some other layouts